core/src/main/java/org/openstack4j/api/OSClient.java

```diff
@@ -278,45 +286,10 @@
      */
     MagnumService magnum();
 
-    /**
-     * Returns the Gbp Service API
-     *
-     * @return the Gbp service
-     */
-    GbpService gbp();
-
-    /**
-     * Returns the Senlin Service API
-     *
-     * @return the Senlin service
-     */
-    SenlinService senlin();
-
-    /**
-     * Returns the Trove Service API
-     *
-     * @return the Trove service
-     */
-    TroveService trove();
-
-    /**
-     * Returns the Barbican Service API
-     *
-     * @return the Barbican service
-     */
-    BarbicanService barbican();
-
-    /**
-     * Returns the DNS Service API
-     *
-     * @return the DNS service
-     */
-    DNSService dns();
-
     /**
      * OpenStack4j Client which authenticates against version V2
      */
-    interface OSClientV2 extends OSClient<OSClient.OSClientV2> {
+    public interface OSClientV2 extends OSClient<OSClient.OSClientV2> {
```

Comment:
Unrelated and unneeded. All declarations in interfaces are public by default.

Reply:
I have reverted all indentation changes and updated the commit. Please review.